### DISCUSSION TITLE
Strimzi mirrormaker Active/Passive  failover and failback strategies with consumer group offset sync

### DISCUSSION THREAD

**Discussion by Veljen** [strimzi/strimzi-kafka-operator#9549] [Q&A] [answered]
Hi All,

we have 2 strimzi cluster configure with mirror maker2  (active/passive) with same topic name. 

cluster A --> ClusterB mirroring and offset consumer group sync in fine

If I failover the traffice from cluster A to cluster B (Producer and consumer). It will work with clusterB and clusterA don't have the replication topic and consumer group offset. If I want to failback to clusterA.  How can I do that? 

I tried deleted the mirrormaker instance from clusterA to clusterB and created one from clusterB to clusterA before the faiback but it the consumergroup offset was different and there were duplicate messages on the topic. 

Please share me the best strategies for the `failback` having the topic replicated and consumergroup offset  sync.

**Comment by scholzj** [comment, marked as answer]
I think you have to delete cluster A and set it up from scratch as well. Mirror Maker 2 has no way to somehow follow-up on the previous state when you reverse the mirroring. So you have to start from a clean state and mirror everything from scratch. (The general expectation for active-passive is disaster recovery. So in many cases the cluster A is anyway already lost.)

**Reply by Veljen** [reply]
Thanks @scholzj
